Source/NSView.m

```diff
@@ -750,10 +750,9 @@ - (void) dealloc
   TEST_RELEASE(_frameMatrix);
   TEST_RELEASE(_boundsMatrix);
   TEST_RELEASE(_sub_views);
-  if (_rFlags.has_tooltips != 0)
-    {
-      [GSToolTips removeTipsForView: self];
-    }
+
+  [GSToolTips removeTipsForView: self];
```

Comment:
Did you test, whether you still need this change? It won't do any harm, maybe slow things a bit down. But what we really should know is whether we addressed the underlying issue.